app/app.rb

```diff
@@ -0,0 +1,77 @@
+require 'sinatra/base'
+require 'sinatra/flash'
+require './app/support/data_mapper_setup'
+
+class Chitter < Sinatra::Base
+  register Sinatra::Flash
+  enable :sessions
+  set :session_secret, 'super secret'
+
+helpers do
+  def current_user
+    @current_user || User.get(session[:user_id])
+  end
+
+  def peeps
+    if Peep.all.length > 0
+      Peep.all(:order => [:created_at.desc])
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)